roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -62,8 +62,8 @@ def __init__(self, *args, **kwargs):
         components = self.get_var("oreg_url", default="").split('/')
         self.registries["oreg"] = "" if len(components) < 3 else components[0]
         self.skopeo_command_creds = ""
-        oreg_auth_user = self.get_var('oreg_auth_user', default='')
-        oreg_auth_password = self.get_var('oreg_auth_password', default='')
+        oreg_auth_user = self._templar.template(self.get_var('oreg_auth_user', default=''))
+        oreg_auth_password = self._templar.template(self.get_var('oreg_auth_password', default=''))
```

Comment:
It's possible that ``self._templar`` could be ``None`` so line 65 and 66 could throw ``AttributeErrors``. Should these read more like:

```python
oreg_auth_user = self.get_var('oreg_auth_user', default='')
oreg_auth_password = self.get_var('oreg_auth_password', default='')
if self._templar is not None:
    oreg_auth_user = self._templar.template(oreg_auth_user) 
    oreg_auth_password = self._templar.template(oreg_auth_password)
```

Reply:
It's possible as far as this particular use case is concerned.  It's probably not actually possible during a run of ansible.  In any case, this is a good suggestion, and I won't have to add that testing glue-code.